initfiles/bash/etc/init.d/hpcc_common.in

```diff
@@ -445,6 +435,30 @@ startCmd() {
       fi
     fi
 
+    limits=(
+      MIN_Hn_nofile="32768"
+      MIN_Hc_core="unlimited"
+      MIN_Hu_nproc="8192"
+      MIN_Hr_rtprio="4"
+      MIN_Hl_memlock="unlimited"
+      MIN_Hs_stack="512" )
```

Comment:
Well it will only be set to 512 if it is higher than the system limits, in which case all the other components would have been running w/ a stack lower than 512.  We could just increase it here though for everything.  Do you have a desired size in mind?